notebooks/supernova_model_sims.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>maybe can you cut the AB Mag at 30 or 32?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/230/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
The wrong output was being displayed for that cell. Fixed now